testcases/PowerNVDump.py

```diff
@@ -791,16 +791,47 @@ def runTest(self):
                 boot_type = self.kernel_crash(crash_type="hmc")
                 self.verify_dump_file(boot_type)
 
+class KernelCrash_KdumpWorkLoad(PowerNVDump):
+
+    # This test verifies kdump/fadump after running ebizzy.
+    # ebizzy url needs to be given in ~/.op-test-framework.conf.
+    # Ex: url=http://liquidtelecom.dl.sourceforge.net/project/ebizzy/ebizzy/0.3/ebizzy-0.3.tar.gz
+
+
+    def runTest(self):
+        if not self.url:
+            raise self.skipTest("Provide ebizzy url in op-test-framework.conf")
+        self.cv_SYSTEM.goto_state(OpSystemState.OS)
```

Comment:
are we not already in OS login state as mentioned in conf file, why explicitly call the OS state ?

Reply:
> @Pavithra1602 Get in sync with @harihare as a major change to this test case is happening
> 
> also the test starts and ends before the dump is trigger, it is same as no load while dump is trigger,, is there a way to trigger dump while parallel ebizzy running in back ground ?

As we are starting ebizy for 60sec and triggering dump in 10sec, ebizzy will still be running.

[console-expect]#./ebizzy -S 60&
./ebizzy -S 60&
[1] 30340
[console-expect]#echo $?
echo $?
0
[console-expect]#ps -ef|grep ebizzy
ps -ef|grep ebizzy
root     30340  7396 99 17:28 hvc0     00:02:54 ./ebizzy -S 60
root     30553  7396  0 17:29 hvc0     00:00:00 grep ebizzy
[console-expect]#echo $?
echo $?
0